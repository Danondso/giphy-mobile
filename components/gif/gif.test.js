import React from 'react';
import Gif from './gif';
import renderer from 'react-test-renderer';

test('renders SearchInput correctly', () => {
    const tree = renderer
      .create(<Gif />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });