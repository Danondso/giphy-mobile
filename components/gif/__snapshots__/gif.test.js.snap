// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SearchInput correctly 1`] = `
<View
  style={
    Object {
      "padding": 12,
    }
  }
>
  <Image
    source={
      Object {
        "uri": undefined,
      }
    }
    style={
      Object {
        "height": 250,
        "width": 250,
      }
    }
  />
</View>
`;
